### ISSUE TITLE
MISSING ALT TAGS - ACCESSIBILITY

### ISSUE THREAD

**Issue by jasnoordhanoa** [jasnoordhanoa/Porfolio#5]
Every image should have an alt text accompanying it that describes the image in case it can not be displayed on their device for any reason.

STEPS TO REPRODUCE:
1- Go to jasdhanoa.ca
2 - Scroll and Click into the sustainafarm project
3 - Right-click on the images on the sustainafarm project page and go to inspect
3 - Check the alt attribute in the HTML

EXPECTED RESULTS:
All images should have descriptive alt text there since it is important for accessibility purposes

ACTUAL RESULT:
Several images in the project pages like in the sustainafarm page are missing alt attributes

CONFIGURATION:
Device: Desktop (MackBookAir)
Browser: Chrome (v.117.0)

SEVERITY:
Medium

PRIORITY:
Medium

TYPE:
Bug

ASSIGNEE:
Jas
